.github/README.md

```diff
@@ -9,6 +9,10 @@ Check our [Website](https://bpessentials.github.io/Docs/) for features, commands
 
 BP:Essentials aims to bring a simplistic yet powerful command suite to Broke Protocol
 
+## How to Compile
+
+1. Change envirorment Variable `BPDIR` to the path of your Broke Protocol Directory: e.g. `BPDIR=E:\Steam\steamapps\common\BROKE PROTOCOL`
```

Comment:
Minor typo (envirorment -> environment), changed wording, and added steps 2 and 3 for clarity 
```suggestion
1. Change the environment variable `BPDIR` to the path of your Broke Protocol game directory: e.g. `BPDIR=E:\Steam\steamapps\common\BROKE PROTOCOL`. Make sure this environment variable is set _before_ running your IDE, for example Visual Studio.
2. Open the `.sln` in your favorite IDE. Suggested IDE is currently Visual Studio 2019 or VSCode (note, for VSCode you may need to create a `tasks.json` file.)
3. Within your IDE, press build to compile the project. Using Visual Studio 2019 (or VSCode if you installed [this](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vs-keybindings) extension), you may also press `CTRL+SHIFT+B` to build.
```